CLAUDE.md

```
@@ -27,6 +27,24 @@ docker build -t mcp/crawl4ai-rag --build-arg PORT=8051 .
 docker run --env-file .env -p 8051:8051 mcp/crawl4ai-rag
 ```
 
+### Database Setup and Migrations
+
+#### Supabase Migration Limitations
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)